packages/@office-iss/react-native-win32/src/Libraries/QuirkSettings/CachingNativeQuirkSettings.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ * @flow strict
+ */
+
+import NativeQuirkStorage from './NativeQuirkSettings';
+import type {Spec as NativeQuirkSettingsSpec} from './NativeQuirkSettings';
+import type {Quirks} from './Quirks';
+import {DefaultQuirks} from './Quirks';
+
+let cachedQuirks: ?Quirks = null;
+
+const CachingNativeQuirkSettings: NativeQuirkSettingsSpec = {
```

Comment:
NitPicking here as we do this all over the place. Just sharing obervation :)

It seems that `CachnigNativeQuirkSettings.js` and `NativeQuickSettings.js` are private implementation detials and the real public surface should be the type `Quirks` in quirks.js and the function `getConstants` in quicksettings.js...

Should this feature not be in it's own package.json to add that level of visbility abstraction?

Reply:
There is a `ReactNativePrivateInterface` thing that aggregates internal libraries that maybe I could add to, to only export the public bits. This is a bit coupled to internals of RN, and our repo is not set up well for multiple flow packages, so a whole separate package for this may not be worth it.